### DISCUSSION TITLE
removal of cpeditor

### DISCUSSION THREAD

**Discussion by bishen28** [cpeditor/cpeditor#1072] [Q&A] [answered]
how to remove cpeditor using linux command?? or there any other way to remove it??

update: i got the my answer.
removed using $ sudo snap remove cpeditor

**Comment by ouuan** [comment, marked as answer]
How did you install it?

**Reply by bishen28** [reply]
installed using :- $ sudo snap install cpeditor --classic
now also removed using $ sudo snap remove cpeditor

**Reply by ouuan** [reply]
> now also removed using $ sudo snap remove cpeditor

So what's the problem?

**Reply by bishen28** [reply]
sorry, i was just figuring out a way to remove it, but wasn't able to remove so i posted the question here, but after some time i got the correct way, so now there not any problem.
